### PR TITLE
Bookstore: add reducers and actions

### PR DESCRIPTION
**Pull request by belmeetmule**
### In this PR, I have completed the following activities:

- [ ] add actions and reducer for book 
- [ ] add actions and reducer for categories
- [ ] configure store for the app

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// Actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```
Comment (Davidosky007) [inline src/redux/books/books.js]: * Kindly use the [ducks' pattern](https://github.com/erikras/ducks-modular-redux) to write the values of your action types ( ie it should look something like this `const ADD_BOOK = bookstore/books/ADD_BOOK` ) as it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Write%20your%20book%27s%20actions%20and%20reducer%20(in%20/src/redux/books/books.js)%20using%20the%20ducks%20pattern.)

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const STATUS_CHECKED = 'STATUS_CHECKED';
```
Comment (Davidosky007) [inline src/redux/categories/categories.js]: * Same thing here Kindly use the ducks' pattern for your action types value as its a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Write%20your%20book%27s%20actions%20and%20reducer%20(in%20/src/redux/books/books.js)%20using%20the%20ducks%20pattern.)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @belmeetmule 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking readme file 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @belmeetmule 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**